src/cpython/jlwrap.jl

```diff
@@ -220,7 +223,10 @@ function _pyjl_deserialize(t::PyPtr, v::PyPtr)
         x = deserialize(io)
         return PyJuliaValue_New(t, x)
     catch e
-        PyErr_SetString(POINTERS.PyExc_Exception, "error deserializing this value")
+        PyErr_SetString(
+            POINTERS.PyExc_Exception,
+            "error deserializing this value. Caught exception $e"
+        )
```

Comment:
If you're printing out the exception, why not also include the trace? e.g.
```suggestion
        PyErr_SetString(
            POINTERS.PyExc_Exception,
            "error deserializing this value. Caught exception $(sprint(showerror, e. catch_backtrace()))"
        )
```

Reply:
good idea!